src/components/tree-item/tree-item.tsx

```diff
@@ -165,6 +166,13 @@ export class TreeItem implements ConditionalSlotComponent {
     this.updateAncestorTree();
   }
 
+  componentDidrender(): void {
+    if (this.hasChildren) {
+      readTask(() => {
+        this.getChildrenContainerHeight();
```

Comment:
Should this be renamed to `setChildrenContainerHeight`?

Reply:
Nice use of `readTask`. @jcfranco should we use read/write task more often? In other cases I've just modified the element at various times but we might be better off using this for perf.

Reply:
Yeah, we could use `readTask`/`writeTask` more to minimize layout thrashing (for the cases we truly need to make changes that would cause this), etc.